src/Bicep.LangServer/Completions/BicepCompletionContextKind.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System;
+
+namespace Bicep.LanguageServer.Completions
+{
+    [Flags]
+    public enum BicepCompletionContextKind
```

Comment:
>BicepCompletionContextKind [](start = 16, length = 26)

Encoding flags in an enum should avoid the property explosion that Roslyn has with this approach.